core/src/main/resources/jenkins/tasks/filters/impl/RetainVariablesLocalRule/help-retainCharacteristicEnvVars.html

```diff
@@ -0,0 +1,5 @@
+<p>
+    When checked, <em>characteristic environment variables</em> will be retained in addition to the variables listed above.
+    These environment variables are job- and build-specific, defined by Jenkins, and are used to identify and kill processes started by this build step.
+    <a href="https://jenkins.io/redirect/spawning-processes/" target="_blank">See the documentation for more details on spawning processes.</a>
```

Comment:
This url appears to be 404

Reply:
Thanks for the reminder. I'll need to add that once this is merged, it'll probably point to https://wiki.jenkins.io/display/JENKINS/ProcessTreeKiller until we have better docs.